src/components/store/ItemHeader/style.module.scss

```diff
@@ -0,0 +1,57 @@
+@use 'src/styles/vars.scss' as vars;
+$indicator-size: 30px;
+$switch-start: 15%;
+$button-roundness: 10px;
+
+.price {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+}
+
+.itemHeaderGroup {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+
+  h1 {
+    font-size: vars.$general-heading-1;
+    font-weight: bold;
+  }
+
+  p {
+    color: var(--theme-text-on-background-1);
+    font-size: vars.$general-heading-4;
+  }
+
+  .error {
+    color: vars.$danger-1;
+  }
+
+  .valid {
+    color: vars.$success-1;
+  }
+
+  .button {
+    border-color: var(--theme-accent-line-2);
+    border-radius: $button-roundness;
+    border-width: 1px;
+    font-size: vars.$input-text-medium;
+    font-weight: bold;
+    height: 40px;
+    width: 200px;
+  }
+
+  .buttonInStock {
+    background-color: var(--theme-primary-2);
+    color: vars.$light-background; // or var(--theme-background);
```

Comment:
because the button background color is blue, white text color is legible in both dark and light mode. but for consistency with the login page's button, I think it should use `var(--theme-text-on-background1)` or whatever it's called, so the button text is black in dark mode

Reply:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/54158686/b005ccd6-99de-41f6-a6e1-72854dc349f7)

Figma says use the `background` color I believe. If there is no functional difference I am good with keeping `var(--theme-text-on-background1)`

Reply:
oh, I think `var(--theme-background)` might be better. I mixed them up, oops

`var(--theme-text-on-background1)` would be black in light mode, white in dark mode (used for text), while `var(--theme-background)`  is the opposite: white in light mode, black in dark mode (used for the page background color)

the button's blue background is *brighter* in dark mode, so the text color should be darker to compensate, so I think using `var(--theme-background)` for the button text would be better

it's kind of confusing; here's the sign in button for reference:

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/93a08080-7689-4266-b7a3-c8e2a26319db) ![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/ae249304-d213-45bb-a6f7-99fd386d9335)